transports/sql/design.md

```diff
@@ -37,6 +37,14 @@ The `ReplyToAddress` column contains the value of `NServiceBus.ReplyToAddress` h
 The `Recoverable` column should always contain the value `1` to ensure wire-level compatibility with transport Version 2 and lower.
 
 
+### Backwards Compatibility
+
+The `CorrelationId`, `ReplyToAddress` and `Recoverable` columns are required for backwards compatibility with version 1 of the [NServiceBus.SqlServer](https://www.nuget.org/packages/NServiceBus.SqlServer) transport
```

Comment:
For which version does the backward compatibility hold true?
- version 2 and lower 
- version 2
- version 1